src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -233,9 +247,24 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
             var primaryRepositories = packageSources.Select(sourceRepositoryProvider.CreateRepository);
 
             var allowPrerelease = Prerelease || (version != null && version.IsPrerelease);
+            
+            DependencyBehavior depBehavior = GetSettingsDependencyBehavior(Settings);
+
+            if (!String.IsNullOrWhiteSpace(DependencyVersion))
+            {
+                // if dependency version specified by command line, try to parse
+                if (!Enum.TryParse<DependencyBehavior>(DependencyVersion, out depBehavior))
```

Comment:
Parse with ignore case set to true
https://msdn.microsoft.com/en-us/library/dd991317(v=vs.110).aspx